index.html

```diff
@@ -436,13 +436,19 @@ <h3>Use Cases and Requirements</h3>
 <!-- requirement list start -->
         <ul>
           <li>
+          <li>
+<a>Credentials</a> represent statements made by an issuer.
+          </li>
 <a>Verifiable credentials</a> represent statements made by an <a>issuer</a> in
 a tamper-evident and privacy-respecting manner.
           </li>
           <li>
 <a>Holders</a> assemble collections of <a>verifiable credentials</a> from
 different <a>issuers</a> into a single artifact, a
-<a>verifiable presentation</a>.
+<a>presentation</a>.
```

Comment:
```suggestion
<a>Holders</a> assemble collections of <a>credentials</a> and/or 
<a>verifiable credentials</a> from different <a>issuers</a> into a single
artifact, a <a>presentation</a>.
```

Reply:
@David-Chadwick -- Since you explicitly noted agreement elsewhere, and not here, I have to ask whether you agree or disagree with this edit, which is not properly highlighted by Github. To be clear, I changed --
```
collections of <a>verifiable credentials</a> from different
```
-- to --
```
collections of <a>credentials</a> and/or <a>verifiable credentials</a> from different
```

Reply:
@David-Chadwick -- This change still appears not to have been made.  Do you disagree with it?

Reply:
@David-Chadwick -- Still waiting on this one.  Other changes (from me and @msporny) have been applied during that wait.

What's wrong with this one?

Reply:
I don't believe that holders assemble credentials into presentations, but only verifiable credentials, as per my original text

Reply:
@David-Chadwick -- Are you saying that Holders **cannot** blend verifiable credentials with [unverifiable] credentials to create a presentation, which they can then choose to make verifiable (or not)?

Are you saying that such Holders must produce two presentations, two artifacts, one [unverifiable] presentation from [unverifiable] credentials which must remain an [unverifiable] presentation, and another [unverifiable] presentation from verifiable credentials which they *may* (but are not required to) transform into a verifiable presentation?

What is the technical basis for this?

If you cannot produce one, or at least point to a normative discussion of this in the VCDM, then I *think* (based on [his comment which was made when this was my only pending suggested change](https://github.com/w3c/vc-data-model/pull/808#pullrequestreview-748598348), but you haven't applied his [simultaneously suggested change](https://github.com/w3c/vc-data-model/pull/808#discussion_r704000248), which you said you were happy to accept, nor @dlongley's [refinement thereof](https://github.com/w3c/vc-data-model/pull/808#discussion_r704704940), either of which I would be OK with) @msporny and I and presumably @dlongley (who typically chimes in when he disagrees with a suggested change and has not done so here) believe my suggestion should be applied.  (@dlongley, @msporny -- Whether I've misinterpreted your position or got it right, please do chime in on this one.)

Reply:
> @David-Chadwick -- Are you saying that Holders **cannot** blend verifiable credentials with [unverifiable] credentials to create a presentation, which they can then choose to make verifiable (or not)?
> 

No I am not saying that. I think it is a rather unusual thing to do because a credential and a self-asserted verifiable credential are equal from a trust perspective aren't they? i.e. you are trusting the holder to make both assertions. Whether either or both of these are then packaged as a presentation or a verifiable presentation makes little difference to the verifier from a trust perspective (providing the transport mechanism guarantees message integrity). So really I think this should be outside the scope of the verifiable credential specification, since it is not dealing with verifiable credentials as we typically think of them.

> Are you saying that such Holders must produce two presentations, two artifacts, one [unverifiable] presentation from [unverifiable] credentials which must remain an [unverifiable] presentation, and another [unverifiable] presentation from verifiable credentials which they _may_ (but are not required to) transform into a verifiable presentation?
> 
> What is the technical basis for this?

What is the reason for having a verifiable credentials standard that deals with unverified credentials, and unverified presentations?

> 
> If you cannot produce one, or at least point to a normative discussion of this in the VCDM, then I _think_ (based on [his comment which was made when this was my only pending suggested change](https://github.com/w3c/vc-data-model/pull/808#pullrequestreview-748598348), but you haven't applied his [simultaneously suggested change](https://github.com/w3c/vc-data-model/pull/808#discussion_r704000248), which you said you were happy to accept, nor @dlongley's [refinement thereof](https://github.com/w3c/vc-data-model/pull/808#discussion_r704704940), either of which I would be OK with) @msporny and I and presumably @dlongley (who typically chimes in when he disagrees with a suggested change and has not done so here) believe my suggestion should be applied. (@dlongley, @msporny -- Whether I've misinterpreted your position or got it right, please do chime in on this one.)

I have not made any changes to my PR at this point in time, because I was letting the discussions complete before I submitted a revised PR incorporating all the suggested changes. When I agreed to your set of changes, I did not agree to the [unverified] credential change, only to the other proposed changes.

Reply:
@TallTed is technically correct, [the best kind of correct](https://www.youtube.com/watch?v=hou0lU8WMgo). This PR is dredging up old issues, we avoided talking about just "credentials" and "presentations" by themselves because of the same sorts of arguments ("this is the VERIFIABLE Credentials specification!")... but it keeps coming up.

You can mix-match "credentials" and "verifiable credentials" in "presentations" and "verifiable presentations", full stop. It might not make sense to do so sometimes, all of the semantics of doing so aren't particularly clear yet, but from a technical basis, you can mix-match all day long. IMHO, that's what makes @TallTed's proposed language correct.

I expect this comment to kick off a "but, what does it mean to mix/match credentials and verifiable credentials together?" -- and that's a valid question, and a separate issue. @TallTed's proposed language is about the fact that you can do so.

Reply:
Ok I will accept all the proposed changes and update the PR.
@TallTed would you like to raise a new issue please on Mixing credentials and VCs in (verifiable) presentation
Kind regards
David

Reply:
> [@David-Chadwick] @TallTed would you like to raise a new issue please on Mixing credentials and VCs in (verifiable) presentation

No, because I see no issue with doing so. I do not foresee a world where Verifiable and Unverifiable Credentials remain pure, each to themselves. I believe that Verifiable Presentations will, more often than not, be comprised of a collection of (claims cherry-picked a/k/a derived from) Verifiable Credentials combined with some bookkeeping/tracking information (a/k/a claims entirely made up by the Holder a/k/a claims cherry-picked from) [Unverifiable] Credentials and/or some bookkeeping/tracking information (a/k/a claims cherry-picked from) Verifiable Credential-Requests issued by the Verifier (whatever form such Verifiable Requests end up taking). 

If you do see issues with such mixing, you are welcome to open github issues on the matter, where I hope you will describe the issue in exquisite detail using the vocabulary we have all agreed on and memorialized in the VCDM and associated docs.

Reply:
I do still await commitment of this suggestion, as well as @dlongley's suggestion below.

For future reference, silent disagreement with a change request, in apparent hopes it will disappear when the rest of a PR is merged, is rather passive-aggressive, and is not conducive to civil discourse. If a PR author disagrees with a change request to their PR, it is incumbent upon them to respond to that request, not by silently resolving the conversation (without committing the request), nor by silently ignoring it; the PR author is expected to engage in dialog with the change requester (and/or others in the relevant community, here being the VCWG and CCG) toward finding a mutually (or at least group) agreeable revision.

(@burnburn @brentzundel @msporny @dlongley -- As chairs and editors, please correct me if I'm wrong in my assertions above.)

Reply:
Please read my earlier post in which I categorically stated "Ok I will accept all the proposed changes and update the PR." So your comment above is uncalled for since I neither ignored your changes nor suggested alternative ones. I have stated that I have accepted them. So please be patient and wait for the updated PR.

Reply:
> Please read my earlier post in which I categorically stated "Ok I will accept all the proposed changes and update the PR." So your comment above is uncalled for since I neither ignored your changes nor suggested alternative ones. I have stated that I have accepted them. So please be patient and wait for the updated PR.

I waited 8 days for response to one change.  Which response I only received after directly asking for such multiple times *and* enlisting others to the same.  Unlike the other change requests which were committed without such delay.

I look forward to your updated PR, which should only require that you click a couple of "Commit suggestion" or "Add suggestion to batch" buttons.

Reply:
[This](https://github.com/w3c/vc-data-model/pull/808/files#r699402481) does not appear to have been applied.

Reply:
I have tried to accept this, but have got into a tangle that I don't know how to untangle. Sorry!